bin/adam-submit

```diff
@@ -22,40 +22,15 @@ SCRIPT_DIR="$(cd `dirname $0`/..; pwd)"
 
 # Get list of required jars for ADAM
 ADAM_JARS=$("$SCRIPT_DIR"/bin/compute-adam-jars.sh)
+ADAM_CLI_JAR=$(echo "$ADAM_JARS" | tail -n 1)
 
+ADAM_JARS=$(echo "$ADAM_JARS" | tail -n +1 | perl -pe 's/\n/,/ unless eof')
```

Comment:
e.g. put a `grep -v <some unique adam cli jar identifier>` in `compute-adam-jars.sh`